src/test/java/org/broadinstitute/hellbender/utils/svd/SVDUnitTest.java

```diff
@@ -1,21 +1,50 @@
 package org.broadinstitute.hellbender.utils.svd;
 
+import htsjdk.samtools.util.Log;
 import org.apache.commons.io.FileUtils;
 import org.apache.commons.lang3.ArrayUtils;
 import org.apache.commons.math3.linear.QRDecomposition;
 import org.apache.commons.math3.linear.RealMatrix;
+import org.broadinstitute.hellbender.engine.spark.SparkContextFactory;
+import org.broadinstitute.hellbender.tools.exome.ReadCountCollection;
+import org.broadinstitute.hellbender.tools.exome.ReadCountCollectionUtils;
+import org.broadinstitute.hellbender.utils.LoggingUtils;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
 import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
 
 import java.io.File;
 import java.io.IOException;
 import java.util.List;
 
-/**
- * A class to hold some static methods useful in unit testing the SVD functionality.
- */
-final public class SVDTestUtils {
+//Note: the SVD tests were not moved to public because they use exome packages and those are in protected for now
+public final class SVDUnitTest extends BaseTest {
+    private static final File TEST_FILE_DIR = new File("src/test/resources/org/broadinstitute/hellbender/tools/exome");
+    private static final File CONTROL_PCOV_FULL_FILE = new File(TEST_FILE_DIR, "create-pon-control-full.pcov");
+    private static final File CONTROL_PCOV_GT_SV = new File(TEST_FILE_DIR, "create-pon-control-full.pcov.gt.singular_values");
+    private static final File CONTROL_PCOV_GT_V = new File(TEST_FILE_DIR, "create-pon-control-full.pcov.gt.v");
+    private static final File CONTROL_PCOV_GT_PINV = new File(TEST_FILE_DIR, "create-pon-control-full.pcov.gt.10.pinv_truncated");
 
-    private SVDTestUtils() {
+    @DataProvider(name="makeSVDers")
+    private Object[][] makeSVDers(){
+        return new Object[][]{
+                {new ApacheSingularValueDecomposer()},
+                {new OjAlgoSingularValueDecomposer()},
+                {new SparkSingularValueDecomposer(SparkContextFactory.getTestSparkContext())}
+        };
+    }
+
+    @Test(dataProvider = "makeSVDers")
+    public void testBasicFunction(final SingularValueDecomposer decomposer){
+        LoggingUtils.setLoggingLevel(Log.LogLevel.INFO);
+        try {
+            final ReadCountCollection rcc = ReadCountCollectionUtils.parse(CONTROL_PCOV_FULL_FILE);
+            final SVD svd = decomposer.createSVD(rcc.counts());
```

Comment:
SVD testing does not need to be coupled with `ReadCountCollection` and pcov files.  Rather than referring to files, I would rely on the following necessary and sufficient assertions.  I will use the notation that M is the original matrix and M = U S V* is the SVD, where S is a matrix with the singular values on its diagonal and zeroes elsewhere

* U is unitary
* V is unitary
* M = U S V*
* pseudoinverse = V S' U* where S' is the transpose of the element-by-element inverse of the non-zero elements of S.

This should work for an arbitrary random matrix and thus you don't need any test files.